tests/Load/graphQLConfirmUser.js

```diff
@@ -0,0 +1,68 @@
+import http from 'k6/http';
+import {ScenarioUtils} from "./utils/scenarioUtils.js";
+import faker from "k6/x/faker";
+import {check} from 'k6';
+import {Utils} from "./utils/utils.js";
+import {MailCatcherUtils} from "./utils/mailCatcherUtils.js";
+
+const utils = new Utils();
+const scenarioName = 'graphqlConfirmUser';
+const scenarioUtils = new ScenarioUtils(utils, scenarioName);
+const mailCatcherUtils = new MailCatcherUtils(utils);
+
+export default function () {
+    confirmUser();
+}
+
+export const options = scenarioUtils.getOptions();
+
+async function confirmUser() {
+    const email = faker.person.email();
+    const userResponse = await createUser(email);
+    const user = JSON.parse(userResponse.body);
+    const id = utils.getGraphQLIdPrefix() + user.id;
+    const mutationName = 'confirmUser';
+
+    let token = null;
+
+    if (userResponse.status === 201) {
```

Comment:
will be better if we will move this check in utils, we need a function which can check it and return boolean

Reply:
not fixed